src/index.css

```diff
@@ -0,0 +1,131 @@
+:root {
+  --white: #fff;
+  --red: #e50914;
+  --grey: #b3b3b3;
+  --dark-grey: #444;
+  --light-grey: #555;
+  --shadow-black: rgba(0, 0, 0, 0.85);
+  --orange: #e87c03;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background-image: url(https://assets.nflxext.com/ffe/siteui/vlv3/02dd3f21-2c34-4989-a48b-827dd9edc446/6fd9c935-5097-4045-9e29-4e1028a6c1fb/PL-pl-20190422-popsignuptwoweeks-perspective_alpha_website_large.jpg);
+
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+    monospace;
+}
+
+
+.login-container__form {
+  background-color: var(--shadow-black);
+  max-width: 15%;
+  margin: 5% auto;
```

Comment:
I wouldn't set margin in percentage, also make the form closer to the top of the screen